precheck/lib/precheck/rule.rb

```diff
@@ -5,8 +5,7 @@
 module Precheck
   VALIDATION_STATES = {
     passed: "passed",
-    failed: "failed",
-    skipped: "skipped"
+    failed: "failed"
```

Comment:
`skipped` wasn't used anywhere. I initially wanted to use it, but opted instead to return nil.